app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>TasteyBites</title>
+    <title>Tasty Bites</title>
```

Comment:
- [ ] I totally adore your styling, fantastic work guys :+1:

Reply:
@VICTOR-CHUKWU 
Thank you so much.